UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/WebhookScenarios.cs

```diff
@@ -0,0 +1,40 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using UKHO.ERPFacade.API.FunctionalTests.Helpers;
+
+namespace UKHO.ERPFacade.API.FunctionalTests.FunctionalTests
+{
+    [TestFixture]
+    public  class WebhookScenarios
+    {
+
+        private WebhookEndpoint webhook { get; set; }
+        private TestConfiguration TestConfig { get; set; }
+
+        private string filePathWebhook;
+
+        [SetUp]
+        public void Setup()
+        {
+            webhook = new WebhookEndpoint();
+            TestConfig = new TestConfiguration();
+            DirectoryInfo dir = Directory.GetParent(Environment.CurrentDirectory).Parent.Parent;
+            filePathWebhook = Path.Combine(dir.FullName, TestConfig.PayloadFolder, TestConfig.WebhookPayloadFileName);
+        }
+        [Test]
+        public void WhenValidEventInnewenccontentpublishedeventoptions_ThenWebhookReturns200OkResponse()
```

Comment:
Method  signature should be public async Task  WhenValidEventInnewenccontentpublishedeventoptions_ThenWebhookReturns200OkResponse()

Reply:
Included in PR#15